stl/inc/complex

```diff
@@ -423,10 +449,18 @@ public:
         return (numeric_limits<float>::max)();
     }
 
+    static _Ty _Abs(_Ty _Left) {
+        return _CSTD fabsf(_Left);

```

Comment:
I think it could be explained somewhere that comparison against zero and negating results if less can produce wrong result. Otherwise someone may be tempted to "optimize" copysign. Possibly somewhere near `_Abs` or `_Copysign` is the base place.

Reply:
Added comments in `_Abs`, `_Copysign` and `_Signbit`